diff --git a/apps/cloud_kee_pass/library/vendors/cryptojs/lib-typedarrays.js b/apps/cloud_kee_pass/library/vendors/cryptojs/lib-typedarrays.js
index a9ae5d4..300cfee 100755
--- a/apps/cloud_kee_pass/library/vendors/cryptojs/lib-typedarrays.js
+++ b/apps/cloud_kee_pass/library/vendors/cryptojs/lib-typedarrays.js
@@ -6,7 +6,7 @@ code.google.com/p/crypto-js/wiki/License
 */
 (function () {
     // Check if typed arrays are supported
-    if (typeof ArrayBuffer != 'function') {
+    if (typeof ArrayBuffer === 'undefined') {
         return;
     }
 
