diff --git a/apps/cloud_kee_pass/library/vendors/jparser.js b/apps/cloud_kee_pass/library/vendors/jparser.js
index 128ec93..adae416 100644
--- a/apps/cloud_kee_pass/library/vendors/jparser.js
+++ b/apps/cloud_kee_pass/library/vendors/jparser.js
@@ -48,11 +48,11 @@ jParser.prototype.structure = {
 	int32: function () { return this.view.getInt32(); },
 	float32: function () { return this.view.getFloat32(); },
 	float64: function () { return this.view.getFloat64(); },
-	char: function () { return this.view.getChar(); },
-	string: function (length) {
+	'char': function () { return this.view.getChar(); },
+	'string': function (length) {
 		return this.view.getString(toInt.call(this, length));
 	},
-	array: function (type, length) {
+	'array': function (type, length) {
 		length = toInt.call(this, length);
 		var results = [];
 		for (var i = 0; i < length; ++i) {
@@ -80,7 +80,7 @@ jParser.prototype.structure = {
 		this.view.seek(this.view.tell() + offset);
 		return offset;
 	},
-	if: function (predicate) {
+	'if': function (predicate) {
 		if (predicate instanceof Function ? predicate.call(this) : predicate) {
 			return this.parse.apply(this, Array.prototype.slice.call(arguments, 1));
 		}
